lms/djangoapps/grades/tests/test_models.py

```diff
@@ -183,29 +183,39 @@ def test_update_grade(self):
         """
         Tests model update, and confirms error when updating a nonexistent model.
         """
-        with self.assertRaises(PersistentSubsectionGrade.DoesNotExist):
-            PersistentSubsectionGrade.update_grade(**self.params)
-        PersistentSubsectionGrade.objects.create(**self.params)
-        self.params['earned_all'] = 12
-        self.params['earned_graded'] = 8
-        PersistentSubsectionGrade.update_grade(**self.params)
-        read_grade = PersistentSubsectionGrade.read_grade(
-            user_id=self.params["user_id"],
-            usage_key=self.params["usage_key"],
-        )
-        self.assertEqual(read_grade.earned_all, 12)
-        self.assertEqual(read_grade.earned_graded, 8)
-
-    @ddt.data(True, False)
-    def test_save(self, already_created):
-        if already_created:
+        with patch('lms.djangoapps.grades.models.log') as log_mock:
```

Comment:
To scope this patch statement to only exactly where it's needed, move it down to the successful call to `update_grade`.  This way, readers of the code can tell where exactly to expect the logging.